### DISCUSSION TITLE
[Feature] filter assets, as we can filter data in tabular models

### DISCUSSION THREAD

**Discussion by aisbergde** [immich-app/immich#6279] [Feature Request]
### The feature

assets and their attributes and related properties are organized in tables in the database, tables are related like in a tabular model (semantic model). This kind of tabular models could be browsed and filtered by table attributes (columns or derived columns, for example year derived from date)
=> I would like to browse my assets as I can browse other tabular models.

I could build this kind of model from the immich tables, for example, in Power BI. But Power BI is not an asset viewer.
The idea could be used, to filter assets in immich in this way.

In a first step, it could just look like in PhotoPrism
![image](https://github.com/immich-app/immich/assets/9020187/c5a1d077-fe39-4775-832d-1b5f7339a800)

In the next step, it could be more flexible by using drag and drop, as it is possible for example in Excel Pivot tables.

Technically, it is a kind of graphical query generator to create WHERE conditions, which could be used by #6277
 

### Platform

- [ ] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
Duplicate of  #3156, which will add search filtering that can accomplish the same thing (but won't be a tabular model).
